experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -38,10 +38,22 @@ def get_filterset_kwargs(self, filterset_class):
 
         return kwargs
 
+    def get_filterset(self, filterset_class):
+        kwargs = self.get_filterset_kwargs(filterset_class)
+
+        # Create a second filterset with the status removed from data
+        # to count experiments with all other filters applied except status
+        kwargs_no_status = kwargs.copy()
+        kwargs_no_status["data"] = kwargs_no_status["data"].copy()
+        kwargs_no_status["data"].pop("status")
+        self.filterset_no_status = filterset_class(**kwargs_no_status)
```

Comment:
Can we just move this into get_context_data? I dont see the point of caching this on the view instance since its only going to be used once.

Reply:
Oh yeah we could but then we'd have to pull a bunch of args/methods into the context method that aren't otherwise there.  I don't think caching it on the instance is a problem?